Packs/CommonScripts/Scripts/GetEndpointData/README.md

```diff
@@ -0,0 +1,71 @@
+| Endpoint.EntityBType.value | The type of the destination of the relationship. | String |
+| Endpoint.EntityBType.source | The vendor from which the type of the destination of the relationship of this endpoint was retrieved. | String |
+| Endpoint.ID.value | The endpoint's id. | String |
+| Endpoint.ID.source | The vendor from which the id of this endpoint was retrieved. | String |
+| Endpoint.IPAddress.value | The endpoint's ip address. | String |
+| Endpoint.IPAddress.source | The vendor from which the ip address of this endpoint was retrieved. | String |
+| Endpoint.Domain.value | The endpoint's domain. | String |
+| Endpoint.Domain.source | The vendor from which the domain of this endpoint was retrieved. | String |
+| Endpoint.MACAddress.value | The endpoint's MAC address. | String |
+| Endpoint.MACAddress.source | The vendor from which the Mac address of this endpoint was retrieved. | String |
+| Endpoint.DHCPServer.value | The DHCP server of the endpoint. | String |
+| Endpoint.DHCPServer.source | The vendor from which the DHCP server of this endpoint was retrieved. | String |
+| Endpoint.OS.value | The endpoint's operating system. | String |
+| Endpoint.OS.source | The vendor from which the OS of this endpoint was retrieved. | String |
+| Endpoint.OSVersion.value | The endpoint's operating system version. | String |
+| Endpoint.OSVersion.source | The vendor from which the OSVersion of this endpoint was retrieved. | String |
+| Endpoint.BIOSVersion.value | The endpoint's BIOS version. | String |
+| Endpoint.BIOSVersion.source | The vendor from which the BIOSVersion of this endpoint was retrieved. | String |
+| Endpoint.Model.value | The model of the machine or device. | String |
+| Endpoint.Model.source | The vendor from which the Model of this endpoint was retrieved. | String |
+| Endpoint.Memory.value | Amount of memory on this endpoint. | Integer |
+| Endpoint.Memory.source | The vendor from which the amount of memory of this endpoint was retrieved. | String |
+| Endpoint.Processors.value | The number of processors. | Integer |
+| Endpoint.Processors.source | The vendor from which the Processors of this endpoint was retrieved. | String |
+| Endpoint.Processor.value | The model of the processor. | String |
+| Endpoint.Processor.source | The vendor from which the Processor of this endpoint was retrieved. | String |
+| Endpoint.IsIsolated.value | The endpoint's isolation status. | String |
+| Endpoint.IsIsolated.source | The vendor from which the IsIsolated of this endpoint was retrieved. | String |
+| Endpoint.Status.value | The endpoint's status. | String |
+| Endpoint.Status.source | The vendor from which the Status of this endpoint was retrieved. | String |
+| Endpoint.Vendor.value | The integration name of the endpoint vendor. | String |
+| Endpoint.Vendor.source | The vendor from which the Vendor of this endpoint was retrieved. | String |
```

Comment:
Should this be "The source from which the vendor of this endpoint was retrieved."